src/mappening/api/models/user.py

```diff
@@ -0,0 +1,38 @@
+from mappening.auth.facebook import facebook_oauth
```

Comment:
If you wanna use models legitly check out [MongoEngine](http://docs.mongoengine.org/guide/querying.html#), I used it in a project once. It basically provides structure and database data verification on each usage of database.

Reply:
Also allows different interface for interacting with Mongo and you can add functions that run everytime you push/pull data from the database